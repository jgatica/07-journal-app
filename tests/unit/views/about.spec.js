import { shallowMount } from '@vue/test-utils'
import AboutView from "@/views/AboutView.vue";

describe('Pruebas sobre AboutView', () => {
  it('debe coincidir con snapshot', () => {
    const wrapper = shallowMount(AboutView)

    expect(wrapper.html()).toMatchSnapshot()
  });
})