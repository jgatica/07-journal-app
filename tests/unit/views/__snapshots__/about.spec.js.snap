// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el AboutView debe coincidir con snapshot 1`] = `
<div class="about">
  <h1>This is an about page</h1>
</div>
`;

exports[`Pruebas en el AboutView debe renderizar el componente correctamente 1`] = `
<div class="about">
  <h1>This is an about page</h1>
</div>
`;
